config/src/config/consensus_config.rs

```diff
@@ -179,42 +179,57 @@ impl Default for ConsensusConfig {
 
             // Voting backpressure is only used as a backup, to make sure pending rounds don't
             // increase uncontrollably, and we know when to go to state sync.
-            vote_back_pressure_limit: 30,
+            // Considering block gas limit and pipeline backpressure should keep number of blocks
+            // in the pipline very low, we can keep this limit pretty low, too.
+            vote_back_pressure_limit: 12,
             pipeline_backpressure: vec![
                 PipelineBackpressureValues {
-                    back_pressure_pipeline_latency_limit_ms: 1000,
+                    back_pressure_pipeline_latency_limit_ms: 800,
```

Comment:
This is the only change I'm a bit worried about. It's not really clear to me whether mainnet would start hitting this or not (at low load).

Reply:
we are counting pipeline latency differently from 1.9, and that accounting gives much lower values. (it counts now from the time block enters execution, to the time it gets commit certificate. so a single round trip, vs two before, on top of counting only pending blocks)

didn't initially want to reduce it, but after seeing how much are the values lower from testnet - https://aptoslabs.grafana.net/goto/UN6bnFFSR?orgId=1 - I think even this might be too high. once we see how it looks on mainnet, we can probably reduce it to ~500ms

Reply:
Can you add a comment here explaining ^ ?

Reply:
I think this explanation is relevant for the diff

I can add explanation to how we measure "pipeline_latency" itself using the new method here

Reply:
Yes, I meant that to be added as a comment.